services/QuillLMS/app/assets/stylesheets/variables.scss

```diff
@@ -204,7 +204,7 @@ $family-monospace: "Roboto-mono", "Monaco", "Inconsolata", monospace;
 }
 
 @mixin display-2xl-semibold {
-  font-weight: semibold;
+  font-weight: 600;
```

Comment:
I noticed that this `semibold` value was not actually getting applied, so I updated them to the correct numeric value